src/v3/src/components/Widget/index.tsx

```diff
@@ -306,6 +315,7 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
 
   const resume = useCallback(async () => {
     await initLanguage();
+    let transaction: IdxTransaction | undefined;
```

Comment:
I see you want to pass `transaction` to `handleError` in the `catch` block. Can you use `idxTransaction` instead and eliminate this pattern of having an uninitialized variable?

Reply:
Same thing above in `bootstrap`

Reply:
Actually there is no need to pass transaction to `handleError`. Removed unnecessary code changes.